.pre-commit-config.yaml

```diff
@@ -3,13 +3,22 @@
   hooks:
   - id: yapf
     language: system
-    files: '^(aiida/control/)|(aiida/cmdline/tests)'
+    files: '^(aiida/control/)|(aiida/cmdline/tests)|(docs/update_req_for_rtd.py)'
 
 - repo: git://github.com/guykisel/prospector-mirror
   sha: b27f281eb9398fc8504415d7fbdabf119ea8c5e1
   hooks:
   - id: prospector
     language: system
     exclude: '^(tests/)|(examples/)'
-    files: '^(aiida/control/)|(aiida/cmdline/tests)'
+    files: '^(aiida/control/)|(aiida/cmdline/tests)|(docs/update_req_for_rtd.py)'
+
+- repo: local
+  hooks: 
+  - id: rtd-requirements
+    name: Requirements for RTD
+    entry: python ./docs/update_req_for_rtd.py --pre-commit
+    language: system
+    files: '^(setup_requirements.py)|(docs/requirements_for_rtd.txt)|(docs/update_req_for_rtd.py)'
```

Comment:
* corrected `setup.py` -> `setup_requirements.py`
* also added the updater script